apps/docs/spec/gotrue_v1_config.yaml

```diff
@@ -11,68 +11,80 @@ info:
   title: 'GoTrue' # {string} A readable name.
   source: 'https://github.com/supabase/gotrue' # {string} Where developers can find the source code.
   bugs: 'https://github.com/supabase/gotrue/issues' # {string} Where developers can file bugs.
-  spec: 'https://github.com/supabase/supabase/blob/master/spec/gotrue_v1_config.yaml' # {string} Where developers can find this spec (to link directly in the docs).
+  spec: 'https://github.com/supabase/supabase/blob/master/docs/spec/gotrue_v1_config.yaml' # {string} Where developers can find this spec (to link directly in the docs).
   description: |
-    A `config.toml` file is generated after running `supabase init`. This file is located in the `supabase` folder under `supabase/config.toml`.
+    In a self-hosted environment, you do not have access to the Auth configuration such as third party OAuth provider settings through the Supabase dashboard. Instead, you configure them through the `docker-compose.yml` file. You can read more about the configuration in the [Self-hosting guide](/docs/guides/self-hosting/docker#configuring-services).
```

Comment:
I have seem numerous people being confused that "self-hosted version of Supabase doesn't support OAuth", because they don't see the configuration on the dashboard. I hope this sentence would help clarify what they have to do to configure them.